main.go

```diff
@@ -77,7 +77,7 @@ func pipe(r io.Reader, w io.WriteCloser) {
 }
 
 // pass a stopChan to the go routine
-func proxy(local io.ReadWriteCloser, remoteAddr *net.TCPAddr, stopChan *chan struct{}) {
+func proxy(local *net.TCPConn, remoteAddr *net.TCPAddr, stopChan *chan struct{}) {
```

Comment:
It doesn't look like this changes the behavior of this function--`*.net.TCPConn` implements `io.ReadWriteCloser`. What is this for?